src/components/events/commonEventFields/versionsTable.tsx

```diff
@@ -6,7 +6,7 @@ import {
 } from '@mui/material';
```

Comment:
`{ background: 'whitesmoke' }` is kinda common, const it

Reply:
How and where? Is there a standard?